app/src/main/java/com/hedvig/app/feature/welcome/WelcomeViewModel.kt

```diff
@@ -1,18 +1,22 @@
 package com.hedvig.app.feature.welcome
 
+import android.app.Application
+import androidx.lifecycle.AndroidViewModel
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.viewModelScope
-import com.hedvig.app.feature.dismissiblepager.DismissiblePagerPage
+import com.hedvig.app.R
+import com.hedvig.app.feature.dismissiblepager.DismissiblePagerModel
 import com.hedvig.app.util.apollo.ThemedIconUrls
 import e
 import kotlinx.coroutines.launch
 
 class WelcomeViewModel(
-    private val welcomeRepository: WelcomeRepository
-) : ViewModel() {
+    private val welcomeRepository: WelcomeRepository,
+    application: Application
+) : AndroidViewModel(application) {
```

Comment:
There's some risk with doing this due to how we work with `Context` and language resolution. I've got a nice strategy for refactoring it, let's do it together right away.